pkg/server/server.go

```diff
@@ -2229,6 +2233,55 @@ func (s *topLevelServer) initJobScheduler(ctx context.Context) error {
 	return nil
 }
 
+// runIdempontentSQLForInitType runs one-time initialization.
+func (s *topLevelServer) runIdempontentSQLForInitType(ctx context.Context, typ serverpb.InitType) error {
+	if typ == serverpb.InitType_NO_INIT {
+		return nil
+	}
+
+	initAttempt := func() error {
+		const defaultApplicationClusterName = "application"
```

Comment:
bikeshed: we can't use `default` since it doesn't work as a bare keyword in places where we use a tenant name, but is there anything shorter we could use?